shared/Constants/PhysicalConstants.F90

```diff
@@ -0,0 +1,80 @@
+module MAPL_PhysicalConstantsMod
+
+   use, intrinsic :: iso_fortran_env, only: REAL64, REAL32
+   use MAPL_MathConstantsMod, only: MAPL_PI_R8, MAPL_PI, MAPL_RADIANS_TO_DEGREES, MAPL_DEGREES_TO_RADIANS
+   implicit none
+
+!=============================================================================
+!BOP
+
+! !MODULE: -- A container module for MAPL physical constants
+
+! !PUBLIC VARIABLES:
+
+   ! Universal Constants
+#if defined(CODATA_CONSTANTS)
```

Comment:
oooh! 😄 

@tclune Should we add a CMake `option()` for enabling this? Default to `OFF` of course?

Reply:
Why is the conditional needed?    Seems like should just always provide these.   Other modules can use `ONLY` to exclude if they have their own.

Reply:
This is related to #42 which is to update the constants to CODATA 2018 values. So, you'd change the Stefan-Boltzmann constant, for example. This is an instant non-zero-diff of course so the thought is we can at least have the new constants here, make them non-default for now, then make the switch and eventually remove the old ones.

Reply:
Ah - I did not recognize the name, so thought it was some obscure GEOS option.  (Well, I suppose it now will be.)   

Don't put the cmake option in the ESMA-cmake layer though.  Keep it all in MAPL.

Reply:
Yeah. I'll work with @Gvilla1000-nasa on this.

Reply:
latest push addresses this.